docs/sources/features/datasources/cloudwatch.md

```diff
@@ -254,6 +254,17 @@ it costs $0.01 per 1,000 GetMetricStatistics or ListMetrics requests. For each q
 issue a GetMetricStatistics request and every time you pick a dimension in the query editor
 Grafana will issue a ListMetrics request.
 
+## Service Quota
+
+AWS defines quotas, or limits, for resources, actions, and items in your AWS account. Depending on the number of queries in your dashboard and the amount of users accessing the dashboard, you may reach the limit for the allowed number of CloudWatch GetMetricData requests per second. Note that quotas are defined per account and per region. If you are using multiple regions or have set up more than one CloudWatch data source to query against multiple accounts, you need to request a quota increase for each account and each region.
```

Comment:
`you are` -> `you're`  

[Style guide](https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions)